cluster_management/src/main/java/com/pinterest/rocksplicator/ConfigGenerator.java

```diff
@@ -83,6 +85,7 @@ public ConfigGenerator(String clusterName, HelixManager helixManager, String con
     this.lastPostedContent = null;
     this.disabledHosts = new HashSet<>();
     this.monitor = monitor;
+    this.enableDumpToLocal = new File("/var/log/helixspectator").exists();
```

Comment:
why only enable write to local when this dir exist?

Reply:
can use it to only enable local dump on demand and also to avoid any potential issues caused by the dir is not created or no access right